Trell.Engine/ClearScriptWrappers/EngineWrapper.cs

```diff
@@ -226,8 +226,11 @@ public ScriptObject CreateJsFile(string filename, string type, byte[] contents)
             object module;
 
             using (var t = Cancel(this.engine, linked, this.currentContext).After(limits.MaxStartupDuration)) {
-                var loadWorkerJs = $"import * as hooks from '{work.WorkerJs}'; hooks;";
-                module = this.engine.Evaluate(docInfo, loadWorkerJs);
+                var loadWorkerJs = $"import('{work.WorkerJs}')";
+                var loadEvaluation = this.engine.Evaluate(work.WorkerJs.ToString(), false, loadWorkerJs);
```

Comment:
```suggestion
                var workerJs = work.WorkerJs.ToString();
                var loadWorkerJs = $"import('{workerJs }')";
                var loadEvaluation = this.engine.Evaluate(workerJs , false, loadWorkerJs);
```

To only allocate the string once.

Reply:
Resolved in commit 98887d3